src/components/budget/budget.service.ts

```diff
@@ -234,11 +225,9 @@ export class BudgetService {
 
     const result = await this.budgetRecordsRepo.readOne(id, { session, view });
 
-    const securedProps = await this.authorizationService.secureProperties(
-      BudgetRecord,
-      result,
-      session,
-    );
+    const securedProps = this.privileges
+      .for(session, BudgetRecord)
+      .secure(result);
```

Comment:
Same here, `securedProps` could just be returned straight up.

Reply:
Yea. I just saw securedProps being deprecated this morning as I was doing docs. Will re-factor this as well